CrPatterns/src/main/java/abstrFabr/builder/Builder.java

```diff
@@ -0,0 +1,12 @@
+package abstrFabr.builder;
+
+import abstrFabr.entity.Person;
+
+/**
+ * Created by Ihar_Rubanovich on 3/2/2017.
+ */
+public abstract class Builder {
```

Comment:
Builder should have several methods (which may have different implementation in concrete builders), and Director should call these methods